### ISSUE TITLE
tn5250j License

### ISSUE THREAD

**Issue by andreig09** [tn5250j/tn5250j#50]
Hi,

My name is Andrei, I am a Telecom Engineer, working in Performance testing for the past 6 years, and now working on developing tools for performance testing.

Currently, we are starting an open source project to make a JMeter plugin in order to test IBM Terminals with protocol tn5250. We have been researching about tn5250 libraries, and  found that this one fits perfect with our needs.

The thing is that JMeter has Apache License 2.0 which it isn't compatible with GPL 2.0 License. Do you think there is any chance you could change your type of license for one compatible with Apache License 2.0?

We found your library really useful for our project and it would be great if we could use it.

If you have any questions about our project and how we intend to use your library, please do not hesitate asking.

Thanks in advance for your time!

Best,

Andrei

**Comment by ryaneberly** [comment]
I agree, and I tried.  see https://github.com/tn5250j/tn5250j/issues/1
IT is pretty much impossible to round up all the original contributors and get their approval.

However, I don't think that prevents you from *calling* tn5250j, just from distributing it WITH Apache licensed software (I'm not a lawyer).
